runtimes/kotlin-1.6/src/main/kotlin/io/openruntimes/kotlin/RuntimeResponse.kt

```diff
@@ -1,24 +1,44 @@
 package io.openruntimes.kotlin
 
-import io.javalin.plugin.json.JsonMapper
+import com.google.gson.GsonBuilder;
+import com.google.gson.Gson;
 
-class RuntimeResponse(
-    var data: Any = "",
-    var statusCode: Int = 200
-) {
-    fun json(data: Map<String, Any?>, statusCode: Int = 200): RuntimeResponse {
-        this.data = data
-        this.statusCode = statusCode
-        return this
+public class RuntimeResponse {
+    fun send(body: String, statusCode: Int, headers: MutableMap<String, String>): RuntimeOutput {
```

Comment:
You can use default parameters and remove the overloaded methods

```suggestion
    fun send(body: String, statusCode: Int = 200, headers: MutableMap<String, String> = mutableMapOf()): RuntimeOutput {
```